pkgs/applications/networking/cluster/terraform-providers/teleport/default.nix

```
@@ -1,17 +1,19 @@
-{ lib, fetchFromGitHub, buildGoModule }:
+{ lib, fetchFromGitHub, buildGoModule, teleport }:
 
 buildGoModule rec {
   pname = "terraform-provider-teleport";
-  version = "7.3.0";
+  version = "8.0.6";
 
   src = fetchFromGitHub {
     owner = "gravitational";
     repo = "teleport-plugins";
     rev = "v${version}";
-    sha256 = "19zn78nn64gc0nm7ycblzi4549a0asql07pfxvrphi6s9fjr5m3y";
+    sha256 = "1rhvpbw4dga256dp2cr5f912d2j7rh8pd1v88dlgq3mmw8n5c7vy";
   };
   vendorSha256 = null;
 
+  nativeBuildInputs = [ teleport ];
```

Doesn't seem to be needed during the build.

```suggestion
  checkInputs = [ teleport ];
```
Also the generic `terraform-provider` build now disables `doCheck` by default so you could do that here and drop the `teleport` input entirely.

---

Thanks, was not aware of `checkInputs`. I left the check phase enabled for now, might disable if it becomes more problematic in the future.